cmd/shortener/main.go

```diff
@@ -1,3 +1,34 @@
 package main
 
-func main() {}
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/FoGezz/go-url-shortener/cmd/shortener/config"
+	"github.com/FoGezz/go-url-shortener/internal/app/handler"
+	"github.com/FoGezz/go-url-shortener/internal/app/storage"
+	"github.com/go-chi/chi/v5"
+)
+
+func main() {
+
+	cfg := config.Config{}
+	cfg.Load()
+	fmt.Println("Run on conf", cfg)
+
+	file, err := os.OpenFile("logs.txt", os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
```

Comment:
Please move it from the main package

Reply:
Since logging is the first topic of the 2nd sprint, I decided to remove logs for now